addons/mail/static/src/core/common/date_section.xml

```diff
@@ -2,9 +2,9 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.DateSection">
-    <div class="o-mail-DateSection d-flex align-items-center fw-bolder w-100" t-attf-class="{{ props.className }}">
+    <div class="o-mail-DateSection d-flex align-items-center fw-bold w-100" t-attf-class="{{ props.className }}">
```

Comment:
Could we take this opportunity to properly reorganize the class list by placing `fw-bold` at the end?